src/commons/EyesUtil.ts

```diff
@@ -104,4 +108,18 @@ export class EyesUtil {
 
     return this;
   }
+
+  /**
+   * Set Debug mode
+   * If set to true the eyes.log file will appear under the root folder.
```

Comment:
why not to log it to reporter?

Reply:
Only 2 options given: 
1. Print logs to console
2. Export logs to file

https://help.applitools.com/hc/en-us/articles/360006914932-How-can-I-enable-Applitools-Eyes-SDK-traces-logs-

On our meeting on applitools we agreed to save to file and add it as a jenkins artifact:
1. Open debug mode and save to file. For investigation.

Reply:
according to applittols documentation : 

> If you want finer control on how the logs are processed, you can provide your own LogHandler implementation instead of StdoutLogHandler.

Reply:
Other then this line there is no documentation on how to do this with wdio-allure. 
However we can attach the file to allure on fail via wdio.conf.js same as we do for browser logs. 

In any case this is a change that needs to be done on Console repo and not related to this change.